pkg/scheduler/framework/plugins/clusteraffinity/scoring.go

```diff
@@ -0,0 +1,31 @@
+package clusteraffinity
+
+import (
+	"context"
+
+	fleetv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+	"go.goms.io/fleet/pkg/scheduler/framework"
+)
+
+// PreScore allows the plugin to connect to the PreScore extension point in the scheduling
+// framework.
+func (p *Plugin) PreScore(
+	_ context.Context,
+	state framework.CycleStatePluginReadWriter,
+	_ *fleetv1beta1.ClusterSchedulingPolicySnapshot,
+) (status *framework.Status) {
+	// TODO not implemented.
+	_, _ = p.readPluginState(state)
```

Comment:
Just curious: the state in this PR is only read at this extension point; I assume it's for linting reasons?

Reply:
yeah, it's only for the linting purpose